sympy/vector/coordsysrect.py

```diff
@@ -196,6 +250,46 @@ def _set_lame_coefficient_mapping(self, curv_coord_name):
             raise ValueError('Wrong set of parameters. Type of coordinate system is defined')
         self._h1, self._h2, self._h3 = coefficient_mapping[curv_coord_name]
 
+    def _transformation_equations(self, equations):
+        """
+        Helper method for set_coordinate_type. It substitute scalar into symbols.
+
+        Parameters
+        ==========
+
+        equations : tuple
+            Tuple of transformation equations
+
+        """
+        from sympy import symbols
+        x, y, z = symbols('x y z')
+        return [eq.subs({x:self.x, y:self.y, z:self.z}) for eq in equations]
+
```

Comment:
We assume here that transformation equations has variables 'x y z'. It's a tricky part, because we need take into consideration that variable name are provided in constructor, by `variable_name` parameter.